uint64 timestamp
float32 Joints1
float32 Joints2